js/index.js

```diff
@@ -19,6 +19,9 @@
   try {
     archive = new DatArchive(window.location)
     archiveInfo = await archive.getInfo()
+    docTitle = document.querySelector('html head title')
```

Comment:
Cool! Right now the title is set as "Photo gallery" in the index.html, but it's probably better to do it this way since someone might fork the app and change the title.

One note: we can set the title this way instead: `document.title = 'new title'`, so lines 22-24 can be much simpler